### ISSUE TITLE
Keyboard Nav doesn't work properly when zoomed in

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#546]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.6

**Browser**
Safari 15

**Problem description**
https://github.com/phetsims/qa/issues/721

When zoomed in, moving between the tabs in the preferences dialog using the arrow keys causes the focus to change AND panning to occur. 

Sorry I missed this the first time around!

**Steps to reproduce**
1. Zoom into sim a bit
2. Navigate to Preferences Menu icon and select
3. Use arrow keys to shift focus between 'General,' 'Visual' and 'Audio'

**Visuals**

![pan](https://user-images.githubusercontent.com/87318828/137769599-3f7e76fd-067a-46fa-bd30-e28a36f0a043.gif)

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.2/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.2 2021-10-11 21:55:48 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/15.0 Safari/605.1.15
Language: en-US
Window: 1440x675
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by jbphet** [comment]
I've updated both the joist branch and the BASE dependencies on the 1.5 branch.  This should be good to go.

**Comment by Nancy-Salpepi** [comment]
Looks good. If I am only zoomed in a bit, there will be no panning when I use the arrow keys to move from tab to tab in the preferences menu. If I am zoomed in a lot, the screen will move to follow the focus as I move from tab to tab. I believe that is the correct behavior.
![panandzoom](https://user-images.githubusercontent.com/87318828/139944163-e50bb154-97fa-4e2a-a1be-37e07bdcf1a6.gif)
